### ISSUE TITLE
BufferOverflowException on Android when I tried to send a message from Android to iOS

### ISSUE THREAD

**Issue by Anees-Iqbal** [bridgefy/bridgefy-android-samples#25]
Android App is crashing with following logs when I tried to send a message from Android to iOS app.

```
Fatal Exception: java.nio.BufferUnderflowException
       at java.nio.Buffer.nextGetIndex(Buffer.java:521)
       at java.nio.HeapByteBuffer.getLong(HeapByteBuffer.java:381)
       at com.bridgefy.sdk.framework.controller.m.f(SourceFile:396)
       at com.bridgefy.sdk.framework.controller.k.a(SourceFile:261)
       at com.bridgefy.sdk.framework.controller.k.a(SourceFile:76)
       at com.bridgefy.sdk.framework.controller.k.a(SourceFile:53)
       at com.bridgefy.sdk.framework.controller.k$1$1.onScanResult(SourceFile:136)
       at android.bluetooth.le.BluetoothLeScanner$BleScanCallbackWrapper$1.run(BluetoothLeScanner.java:580)
       at android.os.Handler.handleCallback(Handler.java:883)
       at android.os.Handler.dispatchMessage(Handler.java:100)
       at android.os.Looper.loop(Looper.java:237)
       at android.app.ActivityThread.main(ActivityThread.java:8167)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:496)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1100)
```

**Comment by kekoyde** [comment]
Thank you for reporting this error, you can help me with Androd CatLog, I am looking for this tag **"BridgefyUtils"**, the problem is related to discovering an iOS device.

**Comment by Anees-Iqbal** [comment]
Hi, I was unable to reproduce the issue for a while but crashes occurred again on Android Apps (Bridgefy version 1.1.28) when the these started to chat with iPad (Bridgefy framework 1.0.1). Logs were the same as pasted above and there was no tag of "**BridgefyUtils**". I observed another detail that it worked fine with App installed on iPad OS **14.1** (Bridgefy 1.0.1) while it crashed for App on iPad OS **13.6**. That crash on Android Apps got fixed only when I updated iPad's OS from 13.6 to 14.1.
